.github/workflows/generate.yml

```diff
@@ -29,10 +29,6 @@ jobs:
           passphrase: ${{ secrets.GPG_PASSPHRASE }}
       - name: Setup
         uses: ./.github/actions/setup
-        with:
-          install_dependencies: "false"
-      - name: Normalize package-lock.json
-        run: npm install
```

Comment:
Why remove this?

Reply:
I thought we don't need it since `action.yml` gets the packages installed. Lmk if I got it wrong.

Reply:
This fixes a know upstream issue with npm where different npm versions can create drift in the lockfile.

Let's use these workflow files unchanged as much as possible aside from what we need for php to work https://github.com/seamapi/javascript-http/tree/main/.github

Reply:
https://github.com/seamapi/php/pull/95/commits/b45cf9203025dc6b677843362835b36095a1b480